.env

```diff
@@ -0,0 +1,4 @@
+ADYEN_API_KEY=AQEyhmfxL4PJahZCw0m/n3Q5qf3VaY9UCJ1+XWZe9W27jmlZiv4PD4jhfNMofnLr2K5i8/0QwV1bDb7kfNy1WIxIIkxgBw==-lUKXT9IQ5GZ6d6RH4nnuOG4Bu//eJZxvoAOknIIddv4=-<anpTLkW{]ZgGy,7
```

Comment:
@kathecorner Please delete the `.env` and regenerate the API key. Never expose them.

Thank you for the PR! Our team will look at it.

Reply:
@kathecorner Better delete the whole branch, and create a new PR without the `.env` file